torch/distributions/wishart.py

```diff
@@ -13,12 +13,9 @@
 _log_2 = math.log(2)
 
 
-def _mvdigamma(x: torch.Tensor, p: int) -> torch.Tensor:
-    assert x.gt((p - 1) / 2).all(), "Wrong domain for multivariate digamma function."
-    return torch.digamma(
-        x.unsqueeze(-1)
-        - torch.arange(p, dtype=x.dtype, device=x.device).div(2).expand(x.shape + (-1,))
-    ).sum(-1)
+def _clamp_above_eps(x: torch.Tensor) -> torch.Tensor:
+    # We assume positive input for this function
+    return x.clamp(min=torch.finfo(x.dtype).eps)
```

Comment:
I'm not sure if the `test_bad_symbolic_registration` test failures are related, but I would think it should be safe to use `torch.finfo` in this helper.

Reply:
Thank you for letting me know. I am new to using torch.finfo. I understand that `torch.finfo` raise a issue in JIT for now.